panther_analysis_tool/analysis_utils.py

```diff
@@ -40,21 +40,31 @@
 )
 
 
+class ClassifiedAnalysis:
```

Comment:
Where did the name `Classified` come from?

Reply:
It's derived from the name of the function that generates this type, `classify_analysis`. Happy to change it if there's something better

Reply:
Just wondering. I have no better suggestion. But it doesn't tell me a whole lot about what the class does, if that helps at all